---
permalink: /research/
title: Research
tagline: Sonal Singhal
tags: [about, research, Singhal]
modified: 27-August-2017
header:
   image: assets/images/research2.jpg
---

Whether walking through urban neighborhoods or going on nature hikes, the diversity of the natural world surrounds us. Our lab researches the origin and maintenance of this species diversity. To answer these questions, we integrate field work, genomics, and experiments.

Our research program focuses on three major aims.

# Macroevolutionary patterns of species diversity
{% include figure image_path="/assets/images/species.png" alt="macro" caption="One reason why species diversity might vary is because speciation rate, or how quickly new species form, also varies. This variation can be seen here across lizards and snakes." %}
When we think of mammals, we might first think about chimps, bears, and elephants. But, more than 60% of mammal species are either some kind of rodent or bat. This is an example of a compelling pattern found across the tree of life: some types of species are more common than others. We are studying the potential causes for this pattern in lizards and snakes.

Work done in collaboration with [Dan Rabosky](http://www.raboskylab.org/) at University of Michigan.

<br>

# The adaptive radiation of desert plants
{% include figure image_path="/assets/images/encelia1.png" alt="this is a placeholder image" caption="(Left) The desert habitat of _E. asperifolia_ and the dune habitat of _E. ventorum_. (Right) Where the two habitats meet, the two species hybridize." %}
_Encelia_ is a radiation of about 20 plant species, most of which live in the open deserts of the southwestern United States and Baja California. Each species of _Encelia_ shows adaptations to the different habitats in which they are found. Interestingly, they evolved these differences even while hybridizing freely with each other -- wherever any two species meet, hybrids form, with no evidence that they are sterile or less fit than their parents. We are researching the evolutionary history of these species and their natural hybrid zones to determine how these species formed and how they are maintained.

Work done in collaboration with [Adam Roddy](http://www.adamroddy.com/) at Florida International University & [Felipe Zapata](http://zapatalab.org/) at UCLA.

<br>

# Comparative species delimitation in lizards
Typically, when researchers try to understand how many species there are (or, delimit species), they do it on a per-species basis. In this research, we are exploring how we can combine data across different species groups to delimit species. Ultimately, we hope that comparing across species will both allow us to understand better how species form and to delimit species in a more robust and equal way. For this work, we focus on over 30 species of North American lizards.

Work done in collaboration with [Adam Leache](http://www.adamroddy.com/) at University of Washington & [Matt Fujita](https://fujitalab.com/) at UT Arlington.

<br>

# The evolution of asexual reproduction in lizards
{% include figure image_path="/assets/images/lizards.png" alt="lizards" caption="The cost of males: an asexually-reproducing population grows twice as fast as a sexually-reproducing population because males cannot bear children." %}
Sexual reproduction has many disadvantages. For example, sexual populations grow half as fast as asexual populations (see above). Given this, you might think asexual reproduction is the norm. But, most multicellular species, including our own, reproduce sexually. Why do so many species reproduce sexually if it is so seemingly disadvantageous? One hypothesis is that sex evolved because sex generates genetic variation, and genetic variation fuels evolution. In fact, some call asexuality an evolutionary dead-end. To test this idea, we are comparing closely-related lizard species that have evolved from producing sexually to asexually.

Work done in collaboration with [Matt Fujita](https://fujitalab.com/) at UT Arlington.


<br>

# Tool development & sharing data
As part of our research, we regularly develop new pipelines for preparing and analyzing high-throughput sequencing data, including transcriptome, target capture, and ddRAD data. We strive to make both our code and our data publicly available via our [GitHub](https://github.com/singhal?tab=repositories) page and online databases like NCBI SRA.